src/App.jsx

```diff
@@ -2,14 +2,16 @@ import React from "react";
 //routing
 import { Routes, Route } from "react-router-dom";
 import routeConfig from "@routes/routeConfig";
+// import Navbar from "./components/navbar/Navbar";
+import Header from './components/header/Header';
 //components
 //TODO: import NAVBAR
 
 const App = () => {
   return (
-    <>
-    {/* //TODO: replcae div component with Navbar*/}
-      <div className="h-[64px] bg-gray-500">NAVBAR COMPONENT</div>
+    <div className="w-screen flex flex-col">
+      {/* <Navbar /> */}
```

Comment:
If it's no longer needed, it might be better to remove it entirely to keep the codebase clean